### ISSUE TITLE
[A11y] Admin sets tab not working properly with keybard and screen reader.

### ISSUE THREAD

**Issue by azle** [OregonDigital/OD2#3735]
### Operating System

MacOS

### Browser

Chrome

### Affected page

Backend

### Admin user?

Yes

### Issue Description

URL Reference: https://oregondigital.org/dashboard?locale=en

<img width="964" height="564" alt="Image" src="https://github.com/user-attachments/assets/ae1c04d3-2ce9-4e4d-8252-1af56d0085e8" />

When I tab to "Admin Sets" using keyboard navigation and VoiceOver. The screen reader indicates that I am on a selected tab 1 of 1 main. Which is correct.

However, it is missing the information if the tab is selected or not selected. It is also missing the loaded content of the tab.

When I tab again instead of taking me to the tab content area it tells me I am leaving the Dashboard and gots to an element in the browser, see the drop down arrow from the screen snap below.

<img width="354" height="37" alt="Image" src="https://github.com/user-attachments/assets/bfca7121-478e-4734-a8c2-027d7c91efe4" />

### Suggested Fix

When I tab to "Admin Sets" it should tell me that it is active. When I click to the tab content area it should tell me I am in the tab main content and read the main tab content header "Adminstrative Sets" > "Recent Activity" then should focus on the first link and should read link "link name" in this case "Oregon Arts Commision".

### Can anybody QA or does this need an expert?

A11y familiarity recommended

### QA Steps

When I tab to Admin sets the screen reader should read the tab label and tell me if it is active or not active. Then tab to the tab content area and it should highlight the main tab content header in this case "Adminstrative Sets" > "Recent Activity" then should focuson the first ink and read the first link name.
